background.js

```diff
@@ -0,0 +1,74 @@
+"use strict";
+
+// Add site to storage
+function addSite(tab, callback) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['addSite' is defined but never used.](https://www.codacy.com/app/tomastaro/notemark/file/1131180199/issues/source?bid=2884821&fileBranchId=2904179#l4)